devices/temp_nix.go

```diff
@@ -28,6 +65,33 @@ func getTemps(temps map[string]int) map[string]error {
 			temps[label] = int(sensor.Temperature)
 		}
 	}
+
+	for name, dev := range smDevices {
+		switch sm := dev.(type) {
+		case *smart.SataDevice:
+			data, err := sm.ReadSMARTData()
+			if err != nil {
+				log.Printf("error getting smart data for %s: %s", name, err)
+				continue
+			}
+			if attr, ok := data.Attrs[194]; ok {
+				val, _, _, _, err := attr.ParseAsTemperature()
+				if err != nil {
+					log.Printf("error parsing temperature smart data for %s: %s", name, err)
+					continue
+				}
+				temps[name] = val
+			}
+		case *smart.NVMeDevice:
+			data, err := sm.ReadSMART()
+			if err != nil {
+				log.Printf("error getting smart data for %s: %s", name, err)
+				continue
+			}
+			temps[name] = int(data.Temperature)
```

Comment:
NVME reports the temperature in Kelvins. I believe this line needs to be adjusted.